docs: icon page (#2488)
marigold-ui/marigold: docs/src/pages/components/icon.mdx

github: https://github.com/marigold-ui/marigold/tree/main/packages/icons/src
edit: https://github.com/marigold-ui/marigold/blob/main/docs/src/pages/components/icon.mdx
switchTheme: true
caption: Component to render Icons.
---

On this page you can learn how [Marigold Icons](/foundation/icons/) should be used.
Icons itself can be used to improve visual interest and get users attention.

> By adding the `size` prop you can change to another size. The default size is `24`.

Reviewer: ```suggestion
The size of the icon can be change via the `size` prop. The default size is `24px`.
```

> The `fill` prop is the second attribute and per default `currentcolor`.

Reviewer: ```suggestion
 The `fill` prop is the second attribute and defaults `currentcolor`.
```

> For a full list of the available Icons, see here [Marigold Icons](/foundation/icons/).

Reviewer: ```suggestion
For a full list of the available Icons go to the [Marigold Icons](/foundation/icons/) page.
```

pnpm add @marigold/icons
```

> To import a specific component you just have to use the name of the icon directly between the `< />` brackets. E.g. see below.

Reviewer: ```suggestion
To import a specific component you just have to use the name of the icon directly between the `< />` brackets. E.g.:
```
